doc/server/api_game/api.yaml

```diff
@@ -303,15 +303,72 @@ paths:
       security:
         - bearerAuth: [ ]
 
+
+  /run/location/persons:
+    get:
+      requestBody:
```

Comment:
For get requests, the request body should not have an impact on the response (see http spec: https://www.rfc-editor.org/rfc/rfc2616#section-9.3). 

Changed this endpoint to use URI parameters here: https://github.com/MANVSim/MANVSim/commit/8843c1062f48dd7004000c7d9e146a6d0005aaac